### PR TITLE
BAH-1104|BAH-790|Buvaneswari|Regression-FormPrivilege table modification in liquibase

### PR DESCRIPTION
**Pull request by buvaneswari-arun**
Liquibase changed

### CODE REVIEW

File: bahmnicore-omod/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -4523,28 +4523,22 @@
...
     </changeSet>
-    <changeSet id="bahmni-core-202011150948" author="Buvaneswari">
+    <changeSet id="bahmni-core-20201210148" author="Buvaneswari">
         <preConditions onFail="MARK_RAN">
             <not><tableExists tableName="form_privilege"/></not>
         </preConditions>
         <comment>Introducing form_privilege table as part of BAH_790 Privilege based access to forms</comment>
         <sql>
-            CREATE TABLE form_privilege (
+            CREATE TABLE form_privilege(
```
Comment (angshu) [inline bahmnicore-omod/src/main/resources/liquibase.xml:4532]: Why aren't we using typical liquibase <createTable> syntax?

<createTable tableName="form_privilege">
            <column name="form_privilege_id" autoIncrement="true" type="INT">
                <constraints primaryKey="true"/>
            </column>
       ....
</createTable>

File: bahmnicore-omod/src/main/resources/liquibase.xml [hunk 2]
```diff
@@ -4523,28 +4523,22 @@
...
             form_privilege_id int(11) NOT NULL AUTO_INCREMENT,
             formId int(11) NOT NULL DEFAULT '0',
             formVersion varchar(50) NOT NULL DEFAULT '0',
             privilegeName varchar(50) NOT NULL DEFAULT '',
             editable tinyint(1) DEFAULT '0',
             viewable tinyint(1) DEFAULT '0',
-            creator int(11) NOT NULL DEFAULT '0',
```
Comment (angshu) [inline bahmnicore-omod/src/main/resources/liquibase.xml:4539]: Why are we removing the "creator" field. This is an important field.

File: bahmnicore-omod/src/main/resources/liquibase.xml [hunk 3]
```diff
@@ -4523,28 +4523,22 @@
...
             date_created datetime NOT NULL,
             date_changed datetime DEFAULT NULL,
-            changed_by int(11) DEFAULT NULL,
```
Comment (angshu) [inline bahmnicore-omod/src/main/resources/liquibase.xml:4542]: Are we deleting the entry or voiding the entry in the form_privilege table? If we are voiding the entry, then we must have the "changed_by" column.

Comment (buvaneswari-arun) [reply bahmnicore-omod/src/main/resources/liquibase.xml:4542]: We are not voiding but deleting the entry.

Comment (angshu) [reply bahmnicore-omod/src/main/resources/liquibase.xml:4542]: We dont need this PR anymore right? please close it

Comment (angshu) [reply bahmnicore-omod/src/main/resources/liquibase.xml:4542]: @buvaneswari-arun ^

**Comment by buvaneswari-arun** [comment]
As the code is already in the master this PR is required to revert the changes to the master. Hence reopening the PR

File: bahmnicore-omod/src/main/resources/liquibase.xml (hunk 3 above)
Comment (buvaneswari-arun) [reply bahmnicore-omod/src/main/resources/liquibase.xml:4542]: BAH-1115 is created for this activity.

**Review by binduak — APPROVED** [review]

**Pull request merged**